tests/config.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+source scripts/include/setup.sh
+
+require_tools helm
```

Comment:
I didn't list it because it is a basic OS command, like `awk`, `grep`, or `sed` and should always be available, at least on all Linux systems.

I guess, given that macOS is moving it from the base OS into an optional commandline utilities package, we could include it. I think there are a few other scripts that make use of it too.

Reply:
Fixed in d120de6